src/test/getRuntimeChoice.spec.ts

```diff
@@ -0,0 +1,71 @@
+import * as sinon from "sinon";
+import { expect } from "chai";
+import * as chai from "chai";
+import * as sinonchai from "sinon-chai";
+import * as utils from "../utils";
+import { getRuntimeChoice } from "../getRuntimeChoice";
+chai.use(sinonchai);
```

Comment:
I didn't see this before - you don't need to do this. It's covered by the mocha bootstrap: https://github.com/firebase/firebase-tools/blob/master/src/test/helpers/mocha-bootstrap.js

Reply:
I need to use sinonchai though, which is different from chaiAsPromised - it breaks the tests when I remove this line.

Reply:
😱 I didn't realize that was a thing! I thought it was `chai-as-promised`. I'm going to remember this exists!

Reply:
Should we move this to the bootstrap anyway, then it's available everywhere?

Reply:
I'm not sure if that's possible - it's either chaiAsPromised or sinon, and some tests will need one while others would need the other. I don't think you can use both, can you?

Reply:
Oh, actually looks like I can do this! Made the changes in the last commit.